src/Controls/src/Core/PlatformPointerEventArgs.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+namespace Microsoft.Maui.Controls;
+
+/// <summary>
+/// Platform-specific arguments associated with the PointerEventArgs
+/// </summary>
+public class PlatformPointerEventArgs
+{
+#if IOS || MACCATALYST
+	/// <summary>
+	/// UIView that has the attached PointerGestureRecognizer
+	/// </summary>
+	public UIKit.UIView Sender { get; }
+
+	/// <summary>
+	/// UIGestureRecognizer attached to the UIView
```

Comment:
Missing cref and sentence should start with "gets" 
https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Property